packages/story-editor/src/constants/zIndex.js

```diff
@@ -16,3 +16,20 @@
 
 // Bring element in front of story details modal
 export const Z_INDEX_STORY_DETAILS = 10;
+
+// Lift the head area from under the canvas, only impacts Karma.
+export const Z_INDEX_HEAD_AREA = 3;
+
+// Floating menu lays on top of side menu
+export const Z_INDEX_CANVAS_SIDE_MENU = 3;
+
+// Floating element menu (a context menu) should be behind other popopups
+export const Z_INDEX_FLOATING_MENU = 4;
+
+// Edit Layer holds footer, popups in footer need to be in front of floating menu
+export const Z_INDEX_EDIT_LAYER = Z_INDEX_FLOATING_MENU + 1;
```

Comment:
If the floating menu is behind the edit menu, using the floating menu in text edit mode is not possible, it should still be in top of it. Perhaps there needs to be another layer for footer / popups?

Reply:
this is a big bummer. we have to completely reorg the flow of the canvas grid and layers to separate out the layers.  Hot fix is here: https://github.com/GoogleForCreators/web-stories-wp/pull/10893 and I made a ticket for follow up. https://github.com/GoogleForCreators/web-stories-wp/issues/10892